scripts/gen_feeds_daily_off.sh

```diff
@@ -11,18 +11,21 @@ export PERL5LIB="../lib:${PERL5LIB}"
 ./remove_empty_products.pl
 ./gen_top_tags_per_country.pl
 #./compute_missions.pl
+
+# Generate the CSV and RDF exports
 ./export_database.pl
-./mongodb_dump.sh /srv/off/html openfoodfacts 10.1.0.102 off
 
 cd $OFF_PUBLIC_DATA_DIR
-gzip < en.openfoodfacts.org.products.rdf > en.openfoodfacts.org.products.rdf.gz
-gzip < fr.openfoodfacts.org.products.rdf > fr.openfoodfacts.org.products.rdf.gz
-
-gzip < en.openfoodfacts.org.products.csv > en.openfoodfacts.org.products.csv.gz
-gzip < fr.openfoodfacts.org.products.csv > fr.openfoodfacts.org.products.csv.gz
+for export in en.openfoodfacts.org.products.csv fr.openfoodfacts.org.products.csv en.openfoodfacts.org.products.rdf fr.openfoodfacts.org.products.rdf; do
+   nice pigz < $export > new.$export.gz
```

Comment:
you should add apt install pigz in Dockerfile as for me it also serve as a documentation of needed packages.

Reply:
@alexgarel Done, thanks!